server/routes/chat.go

```diff
@@ -149,3 +143,177 @@ func userInList(a string, list []string) bool {
     return false
 }
 
+func CreateChatRoom(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+	dsClient := services.Locator.DsClient()
+	userService := services.Locator.UserService()
+	userID := userService.GetCurrentUserID(ctx)
+
+	bodyBytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Printf("Cannot read request body: %v", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	requestedUserID := string(bodyBytes)
+	id := userID + " " + requestedUserID
```

Comment:
I'd stick with the autogenerated chatroom id (you see how image id's are generated for reference)